hazelcast/src/main/java/com/hazelcast/map/impl/mapstore/writebehind/DefaultWriteBehindProcessor.java

```diff
@@ -360,13 +360,16 @@ public void callAfterStoreListeners(Collection<DelayedEntry> entries) {
         boolean result = false;
         Exception exception = null;
         int k = 0;
+        boolean interrupted = false;
         for (; k < RETRY_TIMES_OF_A_FAILED_STORE_OPERATION; k++) {
             try {
                 result = task.run();
+            } catch (InterruptedException ex) {
+                interrupted = true;
```

Comment:
No, i am breaking the loop, but it cant be seen in PR view here. `loop break` is [here] (https://github.com/ahmetmircik/hazelcast/blob/4155235681cca997dab46bd0db745c672ded7776/hazelcast/src/main/java/com/hazelcast/map/impl/mapstore/writebehind/DefaultWriteBehindProcessor.java#L375).